ftw/builder/tests/__init__.py

```diff
@@ -14,3 +15,13 @@ def setUp(self):
         self.portal = self.layer['portal']
         setRoles(self.portal, TEST_USER_ID, ['Manager'])
         login(self.portal, TEST_USER_NAME)
+
+
+class FunctionalTestCase(TestCase):
+
+    layer = BUILDER_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+        login(self.portal, TEST_USER_NAME)
```

Comment:
login should not be necessary - the test user should already be logged in. that's a common mistage :wink: